app/handlers/sessions_help.rb

```diff
@@ -32,8 +69,23 @@ def handle
                            "support for assistance.",
                   offending_inputs: [:email_address])
     end
-    code = run(GeneratePasswordResetCode, user.identity).outputs[:code] \
-      unless user.identity.nil?
-    SignInHelpMailer.sign_in_help(email_addresses.first, code).deliver
+
+    [{
+      user: user,
+      email_addresses: email_addresses
+    }]
+  end
+
+  def matching_users_by_email(email)
+    users = ContactInfo.where(value: email).all.collect(&:user)
+
```

Comment:
This is going to load all the matching contact infos, then fire a select for the user model of each one.  Would be more efficient (and arguably more clear) to: `User.joins(:contact_infos).where({contact_infos: {value: email}})`